scripts/data_files/error.fmt

```diff
@@ -25,8 +25,7 @@
 #include MBEDTLS_CONFIG_FILE
 #endif
 
-#if defined(MBEDTLS_ERROR_C) || defined(MBEDTLS_ERROR_STRERROR_DUMMY)
-#include "mbedtls/error.h"
```

Comment:
Where is error.h included from (maybe platform.h directly)? Do we use any error.h declarations directly in code generated from this template? As a matter of good style, if we directly use declarations from `error.h`, we should explicitly include `error.h` even if some other header pulls it in.

Reply:
`error.c` is generated from this template and the script adds an `error.h` include. I agree that it is the better style to leave it in the template, because it uses declarations from it, but I don't have a strong feelings about it. That is why I removed the include when you asked for it. As I mentioned I don't have strong feelings about it, which version shall it be?

Reply:
I had requested removal from line 113 of error.c, ignoring that that portion was auto generated code. I understand now that both inclusions of error.h were added via code generation: both the template and the module processing bit added error.h. Current shape is OK. To answer my own question, `error.h` is directly included when the code gen adds all the modules and their include files to the top of error.c.